src/bitcoin-cli.cpp

```diff
@@ -315,6 +316,7 @@ class GetinfoRequestHandler: public BaseRequestHandler
         result.push_back(JSONRPCRequestObj("getblockchaininfo", NullUniValue, ID_BLOCKCHAININFO));
         result.push_back(JSONRPCRequestObj("getwalletinfo", NullUniValue, ID_WALLETINFO));
         result.push_back(JSONRPCRequestObj("getbalances", NullUniValue, ID_BALANCES));
+        result.push_back(JSONRPCRequestObj("getmempoolinfo", "[1,10,20,30]", ID_MEMPOOLINFO));
```

Comment:
Updating the code to this solved the error for me.

```suggestion
        UniValue params{RPCConvertValues("generatetoaddress", {"[1,10,20,30]"})};
        result.push_back(JSONRPCRequestObj("getmempoolinfo", params, ID_MEMPOOLINFO));
```